app/uk/gov/hmrc/claimvatenrolmentfrontend/services/StoreVatPostcodeService.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.claimvatenrolmentfrontend.services
+
+import java.time.LocalDate
+
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.Json
+import uk.gov.hmrc.claimvatenrolmentfrontend.repositories.JourneyDataRepository
+import uk.gov.hmrc.claimvatenrolmentfrontend.services.StoreVatRegistrationPostcodeService.{vatRegPostcodeKey}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class StoreVatPostcodeService @Inject()(journeyDataRepository: JourneyDataRepository
```

Comment:
Please could you rename this StoreBusinessPostcodeService (and rename the def and the object too). Just to keep it consistent with the controller name